Update L2_Bridge_schema.graphql

PaulieB14 opened the pull request:
Added descriptions.  *Note: copied from L1.  If I missed any differences please let me know.  But they looked the exact same.

PaulieB14 committed:
Update L2_Bridge_schema.graphql

Added descriptions.  Note copied from L1.  If I missed any differences please let me know.  But they looked the exact same.

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
 }
 
-type MultipleWithdrawalsSettled @entity {
+type ChallengeResolved @entity {
```
This entity shouldn't exist in L2 schema

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
 }
 
-type TransferRootSet @entity {
+type TransferBondChallenged @entity {
```
This entity shouldn't exist in L2 schema

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
 }
 
-type TransferSent @entity {
+type TransferRootBonded @entity {
```
This entity shouldn't exist in L2 schema

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
 }
 
-type TransfersCommitted @entity {
+type TransferRootConfirmed @entity {
```
This entity shouldn't exist in L2 schema

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
   token: String!
 }
 
-type TransfersCommitted @entity {
```
This entity shouldn't be removed

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
-  token: String!
-}
-
-type BonderFee @entity {
```
This entity shouldn't be removed

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
+  "Date unix timestamp"
+  date: Int!
+  "Bridge token asset entity"
   tokenEntity: Token!
```
This entity property shouldn't exist

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
-  date: Int!
-}
+  "Bridge token asset entity"
+  tokenEntity: Token!
```
This entity property shouldn't exist

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
-  token: String!
-}
-
-type RemoveLiquidityImbalance @entity {
```
This entity shouldn't be removed

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
   token: String!
 }
 
-type RemoveLiquidityOne @entity {
```
This entity shouldn't be removed

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
   token: String!
 }
 
-type RemoveLiquidity @entity {
```
This entity shouldn't be removed

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
-  token: String!
-}
-
-type AddLiquidity @entity {
```
This entity shouldn't be removed

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,410 +1,545 @@
   token: String!
 }
 
-type TransferSent @entity {
```
This entity shouldn't be removed

miguelmota reviewed (changes requested):
Thanks for the entity descriptions. The L2 schema is not exactly the same as the L1 schema so copying the schema as is would cause undesired changes. Left some comments

PaulieB14 committed:
Update L2_Bridge_schema.graphql

Sorry I copied L1 over.  I went back and revised the L2 Schema from scratch and added description.  Please advise if any changes are needed.  I added some missing descriptions, but if they need revisions, please let me know.

PaulieB14 commented:
@miguelmota. Sorry I thought L2 was the same.  I went back and started from scratch and added descriptions.  There were some entities that were missing descriptions that I added.  Let me know if revisions are needed.

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,348 +1,588 @@
+  "Deadline timestamp"
   deadline: BigInt!
-
+ƒ
```
there's a random artifact here

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,348 +1,588 @@
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
-  from: String!
```
looks like this got overridden by mistake and should be added

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,348 +1,588 @@
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
-  from: String!
```
looks like this got overridden by mistake and should be added

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -1,348 +1,588 @@
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
-  from: String!
```
looks like this got overridden by mistake and should be added

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -354,57 +594,87 @@ type RemoveLiquidityImbalance @entity {
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
-  from: String!
```
looks like this got overridden by mistake and should be added

miguelmota commented on L2_Bridge_schema.graphql:
```diff
@@ -501,4 +771,8 @@ type Token @entity {
-}
\ No newline at end of file
+}
+
```
these trailing newlines should be removed

miguelmota reviewed (changes requested):
thanks @PaulieB14! see additional minor comments that need to be addressed and then it can be merged

PaulieB14 committed:
Update L2_Bridge_schema.graphql

Made revisions

PaulieB14 committed:
Update L2_Bridge_schema.graphql

made revisions

PaulieB14 commented:
@miguelmota Thanks for the revisions.  I think we are good to go now.

miguelmota commented:
@PaulieB14 looks good! merging
